Documentation/helm-operator.md

```diff
@@ -106,6 +106,7 @@ The following tables lists the configurable parameters of the rook-operator char
 | `csi.enableOMAPGenerator`           | EnableOMAP generator deploys omap sidecar in CSI provisioner pod, to enable it set it to true                               | `false`                                                   |
 | `csi.rbdFSGroupPolicy`              | Policy for modifying a volume's ownership or permissions when the RBD PVC is being mounted                                  | ReadWriteOnceWithFSType                                   |
 | `csi.cephFSFSGroupPolicy`           | Policy for modifying a volume's ownership or permissions when the CephFS PVC is being mounted                               | ReadWriteOnceWithFSType                                   |
+| `csi.cephNFSFSGroupPolicy`           | Policy for modifying a volume's ownership or permissions when the CephNFS PVC is being mounted                               | ReadWriteOnceWithFSType                                   |
```

Comment:
default is File in cephcsi PR https://github.com/ceph/ceph-csi/pull/2948/files#diff-d22e003a7e4dcd8f24ec0228807952dcaacea10d40f0ce6bbc4d4f48e7487e64R11

Reply:
In nfs official repo, it is undefined which should default to RWOnceWithFSType
https://github.com/kubernetes-csi/csi-driver-nfs/blob/master/deploy/v3.1.0/csi-nfs-driverinfo.yaml 
https://kubernetes-csi.github.io/docs/support-fsgroup.html#supported-modes 

@nixpanic wdyt?

Reply:
I would keep the upstream default value for the time being.

Reply:
:+1: